### PR TITLE
Update Featuretools Branch Name

### PR DESCRIPTION
**Pull request by gsheni**
### Pull Request Description
- Change default branch name from `master` to `main`

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -8,9 +8,9 @@ jobs:
     name: PyPI Release
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@main
```
Comment (rwedge) [inline .github/workflows/release.yml:11]: instead of "main" let's use "v2"

File: docs/source/conf.py [hunk 2]
```diff
@@ -63,15 +63,15 @@
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
 
-# The master toctree document.
-master_doc = 'index'
```
Comment (rwedge) [inline docs/source/conf.py:67]: `master_doc` [is a configuration parameter](https://www.sphinx-doc.org/en/1.8/usage/configuration.html#confval-master_doc) in sphinx  so we probably shouldn't change it

File: release.md [hunk 3]
```diff
@@ -28,12 +28,12 @@ Branches on the conda-forge featuretools repo are automatically built and the pa
     ```bash
     git remote add upstream https://github.com/conda-forge/featuretools-feedstock.git
     ```
-4. If you made the fork previously and its master branch is missing commits, update it with any changes from upstream
```
Comment (rwedge) [inline release.md:31]: Unsure if we will be able to rename this branch since it is part of conda-forge, worth looking into

Comment (gsheni) [reply release.md]: It seems that conda-forge defaults to master, and most of the packages use master.
- https://conda-forge.org/docs/maintainer/adding_pkgs.html
- https://github.com/conda-forge

File: docs/source/conf.py (hunk 2 above)
Comment (gsheni) [reply docs/source/conf.py:67]: Fixed

File: .github/workflows/release.yml (hunk 1 above)
Comment (gsheni) [reply .github/workflows/release.yml]: Fixed

File: README.md [hunk 4]
```diff
@@ -88,7 +88,7 @@ Featuretools contains many [different types of built-in primitives](https://prim
 ## Demos
 **Predict Next Purchase**
 
-[Repository](https://github.com/Featuretools/predict_next_purchase/) | [Notebook](https://github.com/Featuretools/predict_next_purchase/blob/master/Tutorial.ipynb)
+[Repository](https://github.com/Featuretools/predict_next_purchase/) | [Notebook](https://github.com/Featuretools/predict_next_purchase/blob/main/Tutorial.ipynb)
```
Comment (gsheni) [inline README.md:91]: Main Branch Added:
https://github.com/Featuretools/predict-next-purchase/tree/main

File: docs/source/guides/performance.rst [hunk 5]
```diff
@@ -113,9 +113,9 @@ When an entire dataset is not required to calculate the features for a given set
...
+An additional example of partitioning data to distribute on multiple cores or a cluster using Dask can be seen in the `Featuretools on Dask notebook <https://github.com/Featuretools/Automated-Manual-Comparison/blob/main/Loan%20Repayment/notebooks/Featuretools%20on%20Dask.ipynb>`_. This approach is detailed in the `Parallelizing Feature Engineering with Dask article <https://medium.com/feature-labs-engineering/scaling-featuretools-with-dask-ce46f9774c7d>`_ on the Feature Labs engineering blog. Dask allows for simple scaling to multiple cores on a single computer or multiple machines on a cluster.
```
Comment (gsheni) [inline docs/source/guides/performance.rst:116]: Main Branch Added:
https://github.com/Featuretools/Automated-Manual-Comparison/tree/main

File: docs/source/guides/performance.rst [hunk 6]
```diff
@@ -113,9 +113,9 @@ When an entire dataset is not required to calculate the features for a given set
...
 
-For a similar partition and distribute implementation using Apache Spark with PySpark, refer to the `Feature Engineering on Spark notebook <https://github.com/Featuretools/predicting-customer-churn/blob/master/churn/4.%20Feature%20Engineering%20on%20Spark.ipynb>`_. This implementation shows how to carry out feature engineering on a cluster of EC2 instances using Spark as the distributed framework. A write-up of this approach is described in the `Featuretools on Spark article <https://blog.featurelabs.com/featuretools-on-spark-2/>`_ on the Feature Labs engineering blog.
+For a similar partition and distribute implementation using Apache Spark with PySpark, refer to the `Feature Engineering on Spark notebook <https://github.com/Featuretools/predicting-customer-churn/blob/main/churn/4.%20Feature%20Engineering%20on%20Spark.ipynb>`_. This implementation shows how to carry out feature engineering on a cluster of EC2 instances using Spark as the distributed framework. A write-up of this approach is described in the `Featuretools on Spark article <https://blog.featurelabs.com/featuretools-on-spark-2/>`_ on the Feature Labs engineering blog.
```
Comment (gsheni) [inline docs/source/guides/performance.rst:118]: Main Branch Added:
https://github.com/Featuretools/predict-customer-churn/tree/main/churn

File: .github/workflows/release.yml [hunk 7]
```diff
@@ -8,9 +8,9 @@ jobs:
     name: PyPI Release
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v2
     - name: PyPI Upload
-      uses: FeatureLabs/gh-action-pypi-upload@master
+      uses: FeatureLabs/gh-action-pypi-upload@main
```
Comment (gsheni) [inline .github/workflows/release.yml:13]: Main Branch added:
https://github.com/FeatureLabs/gh-action-pypi-upload/tree/main

Comment (rwedge) [reply .github/workflows/release.yml:13]: Perhaps we should switch to using tagged releases of this action so that the branch name isn't even a factor

**Pull request merged**